nilearn/regions/region_extractor.py

```diff
@@ -53,7 +54,11 @@ def _threshold_maps_ratio(maps_img, threshold):
     else:
         ratio = threshold
 
-    maps_data = maps.get_data()
+    maps_data = _safe_get_data(maps).copy()
```

Comment:
It's not your fault: we can see elsewhere that the data returned by `_safe_get_data` is copied. But, in my mind, it always returns a copy. Maybe I'm missing a corner case. Does anybody knows if it true or not?